src/pages/payment-methods/index.js

```diff
@@ -32,18 +40,38 @@ const Th = styled.th`
         left: -5px; /* required */
         background-color: var(--color-white);
         z-index: 2;
-
+        width: 128px;
+        padding: 16px 0 16px 16px;
+    }
+    :nth-child(7) {
         p {
             text-align: center;
         }
     }
+    :last-child {
+        width: 50px;
+    }
+`
+const SectionTopContainer = styled(SectionContainer)`
+    @media ${device.tabletL} {
+        padding: 48px 16px 40px;
+    }
+`
+const SectionContentContainer = styled(SectionContainer)`
+    @media ${device.tabletL} {
+        padding: 24px 0 40px;
+    }
+`
+const TopContainer = styled(Container)`
+    @media ${device.tabletL} {
+        width: 100%;
+    }
```

Comment:
We can pass in as responsive props.

Reply:
Have tried this but its not working..so I will stick with this for now..